### PR TITLE
Prepare for 0.7.0 release

### PR DESCRIPTION
**Pull request by lzchen**
Updates all versions of all packages to v0.7.0.

Updates necessary CHANGELOGs.

### CODE REVIEW

File: context/opencensus-context/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+## 0.7.0
+Released 2019-07-31
+
```
Comment (lzchen) [inline context/opencensus-context/CHANGELOG.md]: Was wondering if we should put "-version update" under the notes or leave it as blank?

Comment (reyang) [reply context/opencensus-context/CHANGELOG.md]: I don't have strong opinion, personally I would probably put "bumped version" here.

Comment (lzchen) [reply context/opencensus-context/CHANGELOG.md]: Okay, seeing an empty space is awkward. I will make the change.

**Comment by c24t** [comment]
Does it make sense to release updates to a bunch of packages without any code changes?

**Comment by lzchen** [comment]
@c24t 
You bring up a good point. However, the main concern with having multiple different versions per package is that it's very difficult to maintain as contributors and complicated to debug as users. If the versions for all the packages are changed sooner than later, the better time we will have maintaining the code.

**Comment by c24t** [comment]
LGTM if it doesn't include version updates for packages that haven't changed and don't rely on changes in dependencies.

**Comment by lzchen** [comment]
Fixed in [#741]

**Pull request merged**